### PR TITLE
Track2 sdk RRP : Updated CHANGELOG

### PR DESCRIPTION
**Pull request by cvarsh**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Extensions/SubscriptionResourceExtensionClient.cs [hunk 1]
```diff
@@ -41,6 +47,70 @@ private string GetApiVersionOrNull(ResourceType resourceType)
...
+        /// <item>
+        /// <term>Operation Id</term>
+        /// <description>RecoveryServices_Capabilities</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="location"> Location of the resource. </param>
+        /// <param name="input"> Contains information about Resource type and properties to get capabilities. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response<CapabilitiesResponse>> CapabilitiesRecoveryServiceAsync(AzureLocation location, ResourceCapabilities input, CancellationToken cancellationToken = default)
```
Comment (archerzz) [inline sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Extensions/SubscriptionResourceExtensionClient.cs:66]: The method requires a verb, like `GetCapabilitiesRecoveryService` or `FetchCapabilitiesRecoveryService`. Looks like the root cause is due to the `operation_id` in the swagger definition. Use `override-operation-name` to give it a good name. See here for example: https://github.com/Azure/azure-sdk-for-net/blob/3a7060001946c49af98370638aa7fffadca4c905/sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/autorest.md?plain=1#L58-L60

File: sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Models/CapabilitiesResponse.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.RecoveryServices.Models
+{
+    /// <summary> Capabilities response for Microsoft.RecoveryServices. </summary>
+    public partial class CapabilitiesResponse : ResourceCapabilitiesBase
```
Comment (archerzz) [inline sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Models/CapabilitiesResponse.cs:15]: Avoid using `Response` suffix. Consider use`Result`.

File: sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Models/DnsZoneResponse.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.RecoveryServices.Models
+{
+    /// <summary> DNSZone information for Microsoft.RecoveryServices. </summary>
+    public partial class DnsZoneResponse : DnsZone
```
Comment (archerzz) [inline sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Models/DnsZoneResponse.cs:14]: Avoid `Response` suffix. Recommend to use `Result`.

**Review by archerzz — CHANGES_REQUESTED** [review]
Some names require change. Also check the `README.md`, you need to remove `--prerelease` which blocks the build: https://dev.azure.com/azure-sdk/public/_build/results?buildId=2153573&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=a880e989-7d1a-5c96-a41f-d540b383cc43&l=133

**Comment by archerzz** [comment]
Also please update the version number in `Azure.ResourceManager.RecoveryServices.csproj`

**Comment by cvarsh** [comment]
@hiaga , Can you please check above comments and take care of it as we discussed. Thanks..

**Comment by hiaga** [comment]
@archerzz Pls do not merge this, will raise another PR with changeLog and other changes as discussed offline

**Comment by archerzz** [comment]
closing since #33911 has been merged

**Pull request closed without merging**